### DISCUSSION TITLE
Can iperf be installed on a Linux Redhat ?

### DISCUSSION THREAD

**Discussion by Lucaszzzzz** [esnet/iperf#1270] [Q&A] [answered]
Can iperf be installed on a Linux Redhat ? and what are the requirements for the system version?
  Thanks for your help.

**Comment by bmah888** [comment, marked as answer]
I think RedHat 7 (and newer) include iperf3 in their package distributions, so you could in theory just do `yum install iperf3`. Be advised that RedHat packages are often very old compared to current developments (this is true for many programs, not just iperf3).

If you want to build iperf3 from source, please see the instructions in `README.md` at the top of the `esnet/iperf` repository here on GitHub:  https://github.com/esnet/iperf/blob/master/README.md

**Reply by Lucaszzzzz** [reply]
Thanks a lot.
